### DISCUSSION TITLE
Dark menu bar opencpn

### DISCUSSION THREAD

**Discussion by shlandrews** [bareboat-necessities/lysmarine_gen#402] [Q&A] [answered]
How to enable dark menu bar in opencpn? Very bright at night otherwise.

**Comment by mgrouch** [comment]
It’s easier just to hide it temporarily via settings

**Reply by shlandrews** [reply]
Thanks, I assume this means forcing opencpn in BBN to dark mode priority is more challenging if want right click menus to be dark theme also?

**Reply by mgrouch** [reply]
Might be someone on OpenCPN forums can give you better solution.

**Reply by shlandrews** [reply]
I've seen settings to dark mode before installing opencpn mentioned, tried that by uninstall / dark / install but didn't work. Will raise in forum see if someone found a solution.

**Reply by mgrouch** [reply, marked as answer]
There is a way to change theme for gtk3 applications to dark. OpenCPN is gtk3 app. 
It’s going to apply to all gtk3 apps.
And it’s not a switch by a button but rather by editing a file and restarting ui. 
One of the files mentioned here

https://github.com/bareboat-necessities/lysmarine_gen/blob/bookworm/install-scripts/6-desktop/01-desktopTheming.sh

**Reply by shlandrews** [reply]
That worked, thanks. For anyone else reading -
~/.config/gtk-3.0/settings.ini
add line:
gtk-application-prefer-dark-theme=1
can also add to gtk-4.0

**Reply by mgrouch** [reply]
Glad the issue is solved
